schemas/test/en/test_new_calculated_summary_repeating_blocks.json

```diff
@@ -0,0 +1,393 @@
+                                                            }
+                                                        }
+                                                    ],
+                                                    "text": "Monthly journeys by {transport_name}"
+                                                },
+                                                "mandatory": true,
+                                                "type": "Number"
+                                            }
+                                        ]
+                                    }
+                                }
+                            ],
+                            "summary": {
+                                "title": "transport",
+                                "item_title": {
+                                    "text": "{transport_name}",
+                                    "placeholders": [
+                                        {
+                                            "placeholder": "transport_name",
+                                            "value": {
+                                                "source": "answers",
+                                                "identifier": "transport-name"
+                                            }
+                                        }
+                                    ]
+                                }
+                            }
+                        },
+                        {
+                            "type": "CalculatedSummary",
+                            "id": "calculated-summary-spending",
+                            "title": "We calculate the total monthly expenditure on transport to be %(total)s. Is this correct?",
+                            "calculation": {
+                                "title": "Monthly transport expenditure",
+                                "operation": {
+                                    "+": [
+                                        {
+                                            "source": "answers",
+                                            "identifier": "answer-car"
+                                        },
+                                        {
+                                            "source": "answers",
+                                            "identifier": "transport-cost"
+                                        },
+                                        {
+                                            "source": "answers",
+                                            "identifier": "transport-additional-cost"
+                                        }
+                                    ]
+                                }
+                            }
+                        },
+                        {
+                            "type": "CalculatedSummary",
+                            "id": "calculated-summary-count",
```

Comment:
We should demo this calculated summary being used as a value source. From the card:

> As per the scenario, it must be possible to also reference the total in future blocks for piping/validation/logic (routing/skip/enabled). You can already use calculated summary totals, this is to make sure it still works when using repeating questions

Reply:
Ah I figured it might just be the same as the similar test for dynamic answers, but happy to do for this one too, added another section and functional test to cover 👍